lib/data_update_scripts/20201228194641_append_collective_noun_to_community_name.rb

```diff
@@ -0,0 +1,9 @@
+module DataUpdateScripts
+  class AppendCollectiveNounToCommunityName
+    def run
+      return if SiteConfig.collective_noun_disabled
+
+      SiteConfig.community_name = "#{SiteConfig.community_name} #{SiteConfig.collective_noun}"
```

Comment:
Is there a possibility for collective_noun to be nil?

Reply:
Great question! I just tested this out in the console and it seems like there is a possibility that `collective_noun` could be set to `nil`, but I don't know if this is actually the case for any communities. That being said, I could adjust line 4 to read:
```
return if SiteConfig.collective_noun_disabled || SiteConfig.collective_noun.nil?
```
Or instead, maybe I could add a `nil` guard to line 6 like so:
```
#{SiteConfig&.collective_noun}"
```

Thoughts @maestromac ?

Reply:
That works! you should use `!SiteConfig.collective_noun.present?` instead though since that will account for empty string

Reply:
@maestromac What if I changed it to the code snippet below to account for empty strings, `nil` values, and empty arrays/hashes?
```
SiteConfig.collective_noun.blank?
```

Reply:
great idea!

Reply:
Done! I'll merge/deploy pending a green and passing build. 😄